src/lib/skilling/skills/runecraft.ts

```diff
@@ -14,6 +14,7 @@ export interface Rune {
 	tripLength: number;
 	inputRune?: Bank;
 	inputTalisman?: Bank;
+	nostams?: boolean;
```

Comment:
needs to be camelcased, and it might be better to not do a double-negative `noStams: false`, if you do `stams: true` it would be only a few that need to have it on them too?

Reply:
Good idea, resolved the changes requested